source/help/apps/desktop-changelog.rst

```diff
@@ -1,6 +1,73 @@
 Desktop Application Changelog
 ========================================
 
+Release v4.4.0
+----------------------------
+
+**Download Binaries:** 
+
+- **v4.4.0, release day 2020-02-16**
+
+
+**Breaking Changes** 
+XXX Can be removed if there are none XXX
+
+
+Improvements
+~~~~~~~~~~~~~~~
+
+All Platforms
+^^^^^^^^^^^^^
+
+- Added support for PIV Card Certificate Authentication.
```

Comment:
do we want to specify PIV? maybe we should add that other certificates are also supported?

Reply:
@Willyfrog What other certificates?

Reply:
There's no specific one, this is in fact an improvement in handling client certificates for authentication, we made sure it worked for PIV cards, but it will handle most other client auth certificates